Error message:

```
ccache /home/sebo/Downloads/gcc-arm-none-eabi-7-2018-q2-update/bin/arm-none-eabi-gcc -DBUILD_VERSION=zephyr-v1.13.0-928-g9e676b485f -DKERNEL -DSL_PLATFORM_MULTI_THREADED -DSL_SUPPORT_IPV6 -DUSE_CC3220_ROM_DRV_API -D_FORTIFY_SOURCE=2 -D__LINUX_ERRNO_EXTENSIONS__ -D__ZEPHYR__=1 -I../../../../kernel/include -I../../../../arch/arm/include -I../../../../soc/arm/ti_simplelink/cc32xx -I../../../../soc/arm/ti_simplelink/cc32xx/include -I../../../../soc/arm/ti_simplelink/include -I../../../../boards/arm/cc3220sf_launchxl -I../../../../include -I../../../../include/drivers -Izephyr/include/generated -isystem /home/sebo/Downloads/gcc-arm-none-eabi-7-2018-q2-update/bin/../lib/gcc/arm-none-eabi/7.3.1/include -isystem /home/sebo/Downloads/gcc-arm-none-eabi-7-2018-q2-update/bin/../lib/gcc/arm-none-eabi/7.3.1/include-fixed -I/home/sebo/Downloads/gcc-arm-none-eabi-7-2018-q2-update/arm-none-eabi/include -I../../../../ext/hal/cmsis/Include -I../../../../ext/hal/ti/simplelink/source/ti/devices/cc32xx/. -I../../../../ext/hal/ti/simplelink/source/ti/devices/cc32xx/inc -I../../../../ext/hal/ti/simplelink/source/ti/devices/cc32xx/driverlib -I../../../../ext/hal/ti/simplelink/. -I../../../../ext/hal/ti/simplelink/source -I../../../../ext/hal/ti/simplelink/kernel/zephyr/dpl -I../../../../drivers/wifi/simplelink/. -I../../../../ext/hal/ti/simplelink/source/ti/drivers/net/wifi/porting -I../../../../ext/hal/ti/simplelink/source/ti/drivers/net/wifi/bsd -Og -g -Wall -Wformat -Wformat-security -Wno-format-zero-length -imacros /home/sebo/zephyr/samples/net/wifi/b/zephyr/include/generated/autoconf.h -ffreestanding -Wno-main -mthumb -mcpu=cortex-m4 -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-strict-overflow -Wno-pointer-sign -Wno-unused-but-set-variable -fno-reorder-functions -fno-defer-pop -Werror=implicit-int -Wpointer-arith -ffunction-sections -fdata-sections -mabi=aapcs -march=armv7e-m -std=c99 -MD -MT zephyr/CMakeFiles/offsets.dir/arch/arm/core/offsets/offsets.c.obj -MF zephyr/CMakeFiles/offsets.dir/arch/arm/core/offsets/offsets.c.obj.d -o zephyr/CMakeFiles/offsets.dir/arch/arm/core/offsets/offsets.c.obj   -c /home/sebo/zephyr/arch/arm/core/offsets/offsets.c
In file included from ../../../../kernel/include/kernel_structs.h:10:0,
                 from /home/sebo/zephyr/arch/arm/core/offsets/offsets.c:26:
../../../../include/kernel.h: In function 'k_enable_sys_clock_always_on':
../../../../include/kernel.h:1677:10: error: 'ENOTSUP' undeclared (first use in this function)
  return -ENOTSUP;
          ^~~~~~~
../../../../include/kernel.h:1677:10: note: each undeclared identifier is reported only once for each function it appears in

```

--- a/drivers/wifi/simplelink/CMakeLists.txt
+++ b/drivers/wifi/simplelink/CMakeLists.txt
@@ -4,7 +4,6 @@ if(CONFIG_WIFI_SIMPLELINK)
     $ENV{ZEPHYR_BASE}/ext/hal/ti/simplelink/kernel/zephyr/dpl
     $ENV{ZEPHYR_BASE}/ext/hal/ti/simplelink/source
     $ENV{ZEPHYR_BASE}/ext/hal/ti/simplelink/source/ti/drivers/net/wifi/porting
-    $ENV{ZEPHYR_BASE}/ext/hal/ti/simplelink/source/ti/drivers/net/wifi/bsd
     )
   zephyr_sources(
     simplelink_support.c

